packages/@aws-cdk/aws-ecs/test/cluster.test.ts

```diff
@@ -2228,9 +2226,9 @@ describe('cluster', () => {
           Ref: 'providerD3FF4D3A',
         },
       ],
-      DefaultCapacityProviderStrategy: [
-        { CapacityProvider: 'FARGATE', Base: 10, Weight: 50 },
-      ],
+      // DefaultCapacityProviderStrategy: [
+      //   { CapacityProvider: 'FARGATE', Base: 10, Weight: 50 },
+      // ],
```

Comment:
Did you mean to leave this commented? We should check for it in the test.

Reply:
This is what we discuss over slack https://amzn-aws.slack.com/archives/D03UQPQ3HK6/p1671134106074499
that without lazy list the defaultCapacityProvider strategy is not updating in capacity provider association.